src/api/unsplash-api.ts

```diff
@@ -0,0 +1,29 @@
+import { QueryFunction } from '@tanstack/react-query';
+import { SearchParamsValues } from 'context/SearchParamsContext';
+import { Image, SearchResponse } from 'types/unsplash-types';
+
+const baseUrl = 'https://api.unsplash.com';
+
+export const getPhotos: QueryFunction<
+  SearchResponse | Image[],
+  [string, SearchParamsValues],
+  number
+> = async ({ pageParam, queryKey }) => {
+  const isAnyParamPassed = Boolean(Object.values(queryKey[1]).reduce((a, v) => a + v, ''));
+  const response = await fetch(
```

Comment:
it's really hard to read it
try to use `new UrlSearchParams()`

Reply:
done